packages/browser-sdk/src/toolbar/Switch.tsx

```diff
@@ -0,0 +1,50 @@
+import { Fragment, h } from "preact";
+
+interface SwitchProps extends h.JSX.HTMLAttributes<HTMLInputElement> {
+  isOn: boolean;
```

Comment:
I would probably call this `checked` to match HTML and Chakra (v3) naming-conventions.

Reply:
makes sense to rename it, done 👍